cameras:
  front_left_custom: [Scene, DepthPlanner]
  front_right_custom: [Scene]
  front_center_custom: [Scene]
lidar: [LidarCustom]